pkg/apiutil/util.go

```diff
@@ -41,22 +41,48 @@ type WatchEventMessage_PeerEvent struct {
 	Peer Peer
 }
 
+// ListPathRequest is used by server.ListPath API
+type ListPathRequest struct {
+	TableType      api.TableType
+	Name           string
+	Family         bgp.Family
+	Prefixes       []*LookupPrefix
+	SortType       api.ListPathRequest_SortType
+	EnableFiltered bool
+}
+
+type LookupOption uint8
+
+const (
+	LOOKUP_EXACT LookupOption = iota
+	LOOKUP_LONGER
+	LOOKUP_SHORTER
+)
+
```

Comment:
Some const values in api are directly used. Some are redefined; api.LOOKUP_EXACT is one and this defines LOOKUP_EXACT to zero? I prefer them to be consistent.